pages/docs/manual/latest/record.mdx

```diff
@@ -125,6 +182,44 @@ var meNextYear = {
 
 **Note**: spread cannot add new fields to the record value, as a record's shape is fixed by its type.
 
+### Optional Fields
+Updating an optional field in the scenario above works just like any other field provided you want to set a new value. However, if you want to _remove_ the value of that field, you'll need to use a special syntax. Note that to remove a field here means to set its value to `None`. It looks like this:
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res example
+type person = {
+  age: int,
+  name?: string
+}
+
+let me = {
+  age: 123,
+  name: "Hello"
+}
+
+let withoutName = {
+  ...me,
+  name: ?None
```

Comment:
Somewhere the info should be conveyed that this is not specific to `None` and any expression of option type can go there.